### ISSUE TITLE
White textures when trying to render a GUI Window/LineEdit

### ISSUE THREAD

**Issue by Bambofy** [xamarin/urho#331]
![testingsharp desktop_2018-05-11_14-00-54](https://user-images.githubusercontent.com/1304898/39925480-c3f8e55a-5523-11e8-94fc-c368726cb8ef.png)

**Comment by Bambofy** [comment]
![testingsharp desktop_2018-05-11_14-01-33](https://user-images.githubusercontent.com/1304898/39925497-d3d8a0c8-5523-11e8-91ed-80f3b0a6d2ad.png)

**Comment by Bambofy** [comment]
``
            var window = new Window();
            UI.Root.AddChild(window);

            // Set Window size and layout settings
            window.SetMinSize(384, 192);
            window.SetLayout(LayoutMode.Vertical, 6, new IntRect(6, 6, 6, 6));
            window.SetAlignment(HorizontalAlignment.Center, VerticalAlignment.Center);
            window.Name = "Window";

**Comment by hwd71** [comment]
Yeah, I had that.. you have to manually load the style xml, default or not.
